api/config/cdr_versions_test.json

```diff
@@ -11,7 +11,7 @@
     "publicDbName": "public20180920"
   },
   {
-    "name": "New Test Registered CDR",
```

Comment:
Note that doing this should cause the old CDR version row with this name to get deleted... please confirm that works after the push.

Reply:
Changed it back. Just thought it would be good if it is consistent.

Reply:
Darn. I wanted to see what broke :)

Reply:
OK. We don't support renaming CDR versions right now. For the time being, I think it's OK to have this inconsistency.

Reply:
(BTW, it's easy enough to hack the DB and then change the file to match in a follow-up.)